scripts/docker/Dockerfile

```diff
@@ -1,21 +1,79 @@
-FROM debian:9.8
+## LAYER base OS
+FROM debian:9.11 as os-base
 
 RUN rm /bin/sh && ln -s /bin/bash /bin/sh
 
+# Enable contributor package sources
+ARG DEBIAN_FRONTEND=noninteractive
 RUN sed -i 's/stretch main/stretch main contrib/' /etc/apt/sources.list
+RUN apt-get update && apt-get install --assume-yes apt-utils
 
-ENV DEBIAN_FRONTEND=noninteractive
+# General build niceties
+RUN apt-get update && apt-get install --assume-yes \
+	binutils \
+	bvi \
+	bzip2 \
+	coreutils \
+	git \
+	python3 \
+	rsync \
+	strace \
+	sudo \
+	tree \
+	unzip \
+	vim \
+	wget \
+	xz-utils \
+	zip \
+""
+
+# Install nibuild dependencies
+RUN dpkg --add-architecture i386
+RUN apt-get update && apt-get install --assume-yes \
+	libc6-i686:i386 \
+	libperl5.24:i386 \
+	libstdc++6:i386 \
+""
```

Comment:
Is this needed in this version of the Dockerfile? Given that you have the Azdo version, that seems like the right place for this.